nodes/parachain/src/chain_spec/peregrine.rs

```diff
@@ -39,12 +39,12 @@ use crate::chain_spec::{get_account_id_from_seed, get_from_seed, get_properties,
 pub type ChainSpec = sc_service::GenericChainSpec<GenesisConfig, Extensions>;
 
 pub fn make_dev_spec(id: ParaId) -> Result<ChainSpec, String> {
-	let properties = get_properties("KILT", 15, 38);
+	let properties = get_properties("PILT", 15, 38);
 	let wasm = WASM_BINARY.ok_or("No WASM")?;
 
 	Ok(ChainSpec::from_genesis(
-		"KILT Local",
-		"kilt_parachain_local_testnet",
+		"KILT Peregrine Local",
```

Comment:
This change has to be applied to the Polkadot Apps in order to connect it with the styling.

Reply:
This is only for local testing. So it's no problem if we break it